Microsoft/WebTemplateStudio
# App Service - Deploy server folder only

This change is required for deploying the server folder only.

Changes included:
1. Moved the .deployment file from the root to the server folder
2. Set the appService.deploySubpath to the server folder

**To test:**
1. Launch the extension in preview mode
2. Add App Service
3. Once App Service is deployed:
    - check the location of the .deployment file
    - check that `.vscode/settings.json` contains `appService.deploySubpath: "server"`
5. Do `npm install` and `npm build` (this will be automated in another PR)
4. Do a Ctrl + Shift + P, type in "Azure App Service: Deploy to Web App..."
5. Once deployment is done, click on "Browse website" and make sure that it is running properly
